.github/workflows/custard-run.yaml

```diff
@@ -0,0 +1,166 @@
+        with:
+          command: npx gts lint
+          paths: ${{ needs.affected.outputs.paths }}
+      - name: Check success
+        uses: GoogleCloudPlatform/cloud-samples-tools/actions/steps/update-check@v0.2.3
+        with:
+          check: ${{ steps.in_progress.outputs.check }}
+          status: success
+      - name: Check failure
+        uses: GoogleCloudPlatform/cloud-samples-tools/actions/steps/update-check@v0.2.3
+        if: failure()
+        with:
+          check: ${{ steps.in_progress.outputs.check }}
+          status: failure
+
+  test:
+    if: needs.affected.outputs.paths != '[]'
+    needs: affected
+    runs-on: ubuntu-latest
+    timeout-minutes: 120 # 2 hours hard limit
+    permissions:
+      statuses: write
+    strategy:
+      fail-fast: false
+      matrix:
+        path: ${{ fromJson(needs.affected.outputs.paths) }}
+    continue-on-error: true
+    env:
+      GOOGLE_SAMPLES_PROJECT: long-door-651
+      GOOGLE_SERVICE_ACCOUNT: kokoro-system-test@long-door-651.iam.gserviceaccount.com
+    steps:
+      - name: Check queued
+        uses: GoogleCloudPlatform/cloud-samples-tools/actions/steps/create-check@v0.2.3
+        id: queued
+        with:
+          sha: ${{ github.event.workflow_run.head_sha || inputs.ref || github.sha }}
+          name: (experimental) Custard CI / ${{ github.job }} (${{ matrix.path }})
+          job-name: ${{ github.job }} (${{ matrix.path }})
+          if: ${{ !!github.event.workflow_run }}
+      - name: Setup Custard
+        uses: GoogleCloudPlatform/cloud-samples-tools/actions/steps/setup-custard@v0.2.3
+        with:
+          path: ${{ matrix.path }}
+          ci-setup: ${{ toJson(fromJson(needs.affected.outputs.ci-setups)[matrix.path]) }}
+          project-id: ${{ env.GOOGLE_SAMPLES_PROJECT }}
+          workload-identity-provider: projects/1046198160504/locations/global/workloadIdentityPools/github-actions-pool/providers/github-actions-provider
+          service-account: ${{ env.GOOGLE_SERVICE_ACCOUNT }}
+      - name: Check in_progress
+        uses: GoogleCloudPlatform/cloud-samples-tools/actions/steps/update-check@v0.2.3
+        id: in_progress
+        with:
+          check: ${{ steps.queued.outputs.check }}
+          status: in_progress
+      - name: Run tests for ${{ matrix.path }}
+        run: |
+          timeout ${{ fromJson(needs.affected.outputs.ci-setups)[matrix.path].timeout-minutes }}m \
+            make test dir=${{ matrix.path }}
+      - name: Check success
+        uses: GoogleCloudPlatform/cloud-samples-tools/actions/steps/update-check@v0.2.3
+        with:
+          check: ${{ steps.in_progress.outputs.check }}
+          status: success
+      - name: Check failure
+        uses: GoogleCloudPlatform/cloud-samples-tools/actions/steps/update-check@v0.2.3
+        if: failure()
+        with:
+          check: ${{ steps.in_progress.outputs.check }}
+          status: failure
+      # - name: Upload test results for FlakyBot workflow
```

Comment:
I think we can remove the FlakyBot comments: we're not using it anymore and wouldn't upload xml files if we started using it again. (we probably also need a follow up PR for the other commented out FlakyBot stuff, I can do that)

Reply:
Sure, will remove this.